run_release.py

```diff
@@ -664,9 +686,55 @@ def execute_command(command):
     execute_command(f"find {destination} -type f -exec chmod 664 {{}} \\;")
 
 
+def start_build_of_source_and_docs(db: DbfilenameShelf) -> None:
+    # Get the git commit SHA for the tag
+    commit_sha = subprocess.check_output(
+        ["git", "rev-list", "-n", "1", db["release"].gitname],
+        cwd=db["git_repo"]
+    ).decode().strip()
+
+    # Get the owner of the GitHub repo (first path segment in a 'github.com' remote URL)
+    # This works for both 'https' and 'ssh' style remote URLs.
+    origin_remote_url = subprocess.check_output(
+        ["git", "ls-remote", "--get-url", "origin"],
+        cwd=db["git_repo"]
+    ).decode().strip()
+    match = re.match(r"github\.com/([^/]+)/", origin_remote_url)
+    if not match:
+        raise ReleaseException(f"Could not parse GitHub owner from 'origin' remote URL: {origin_remote_url}")
+    origin_remote_github_owner = match.group(1)
+
+    # We ask for human verification at this point since this commit SHA is 'locked in'
+    print()
+    print(f"Go to https://github.com/{origin_remote_github_owner}/cpython/commit/{commit_sha}")
```

Comment:
These are extra steps for ensuring that the commit SHA isn't on a fork and that there haven't been any unexpected upstream changes to the fork since the local commit.